### ISSUE TITLE
Threat detected by Windows Security: AndroidOS/ZkarletFlash

### ISSUE THREAD

**Issue by ericssimpson** [ngrok/ngrok#61]
Duplicate of this issue [#60 948](https://github.com/inconshreveable/ngrok/issues/948).

![image](https://github.com/user-attachments/assets/5febb8f1-11ba-417a-bcb6-63a24f1473f4)

Will submit as a false positive.

**Comment by ericssimpson** [comment]
Submitted and awaiting reclassification.

https://www.microsoft.com/en-us/wdsi/submission/4e5dbf24-0436-47ed-822b-ebc751a2fb1b

**Comment by russorat** [comment]
Hi @ericssimpson , thank you so much for submitting this!

**Comment by russorat** [comment]
@ericssimpson i can't see the link you pasted, but did we get re-classified?

**Comment by ericssimpson** [comment]
I don't believe any action was taken on it by a manual reviewer, my submission history appears to also be empty (despite the retention being 30 days long). Looks like the request for review expired from my end.

**Comment by ericssimpson** [comment]
@russorat 

I don't believe any action was taken on it by a manual reviewer, my submission history appears to also be empty (despite the retention being 30 days long). Looks like the request for review expired from my end.

**Comment by russorat** [comment]
This is a false positive and we've submitted our binaries for review with Microsoft. We've also released a few new versions of the ngrok agent since this issue was opened. Any new binaries have been submitted for analysis by microsoft. If you are still having issues, please open a new issue with the ngrok agent version and architecture and we will investigate.

**Comment by begrafx** [comment]
I've received a hit on my Antivirus software twice now, each following a `brew upgrade` command on my mac. I've left further info in a response to #85
